docs/content/operator/common/archives.mdx

```diff
@@ -81,7 +92,6 @@ state-archive-read-config:
       # Use iota-mainnet-archive for mainnet
       # Notice there is no archive bucket setup for testnet in GCS
       bucket: "iota-<testnet|mainnet>-archive"
-      # Use your gcloud service account credentials
       google-service-account: "</path/to/service/account/credentials>"
```

Comment:
does that still exist?

Reply:
Yes looks like it exists in the `pub strcut ObjectStoreConfig` [here](https://github.com/iotaledger/iota/blob/docs/restructure-node-setups-and-docs/crates/iota-config/src/object_storage_config.rs#L78) as follows

```rust
   #[serde(skip_serializing_if = "Option::is_none")]
    #[arg(long)]
    pub google_service_account: Option<String>,
```